### PR TITLE
refactor(platform): BIG-31201 add circleci

### PR DESCRIPTION
**Pull request by repzy**
#### What?

Add CircleCi config.

### CODE REVIEW

**Comment by serroba** [comment]
I see you still have Travis here. We should be removing it from the list

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+orbs:
+  ci: bigcommerce/internal@volatile
+  php: bigcommerce/internal-php@volatile
+
+jobs:
+  tests-unit:
+    parameters:
+      php-version:
+        type: string
+        default: php-7.3
```
Comment (serroba) [inline .circleci/config.yml:12]: can you add also 7.4? we use this project on both versions.

Comment (repzy) [reply .circleci/config.yml:12]: added.

**Comment by repzy** [comment]
@KenZSyd could you help me with error I got in circleci?

**Comment by KenZSyd** [comment]
> @KenZSyd could you help me with error I got in circleci?

Maybe change the versions to [ "7.3-fpm", "7.4-fpm" ] ?

**Review by serroba — APPROVED** [review]
Nice!

**Pull request merged**